integration-test/common/src/test/java/org/apache/uniffle/test/QuorumTest.java

```diff
@@ -316,6 +317,65 @@ public void rpcFailedTest() throws Exception {
     assertEquals(readClient.readShuffleBlockData(), null);
   }
 
+  @Test
+  public void testRpcRetryLogic() {
```

Comment:
Could you put this test case into another class?

Reply:
In `QuorumTest`, there is `rpcFailedTest` already. I think this is the best place to put `testRpcRetryLogic`.